src/conversions.jl

```diff
@@ -434,6 +434,46 @@ julia> AstroTime.Epochs.ut1tt(ut1.jd1, ut1.jd2, AstroTime.Epochs.deltat(ut1))
      date, date1
 end
 
+@inline function jd2cal(jd1, jd2)
+    dj = jd1 + jd2
+    if dj < JD_MIN || dj > JD_MAX
+        error("Date is not acceptable")
+    end
+
+    if jd1 >= jd2
+        date = jd1
+        date1 = jd2
+    else
+        date = jd2
+        date1 = jd1
+    end
+
+    date1 -= 0.5
+
+    f1 = mod(date, 1.0)
+    f2 = mod(date1, 1.0)
+    f = mod(f1 + f2, 1.0)
+    if f < 0.0
+        f += 1.0
+    end
+    d = round(date-f1) + round(date1-f2) + round(f1+f2-f)
+    jd = round(d) + 1.
+
+    l = jd + 68569.
+    n = (4. * l) / 146097.
```

Comment:
You need to explicitly use integer division on the Julia side, e.g. `n = (4 * l) ÷ 146097` (`÷` tab-completes from `\div` in editors with Julia plugins) or `n = div((4 * l), 146097)`. I would prefer the unicode operator.